jobs/haproxy/spec

```diff
@@ -580,6 +584,13 @@ properties:
       cidr_whitelist:
       - 172.168.4.1/32
       - 10.2.0.0/16
+    ha_proxy.cidr_proxies:
```

Comment:
this list should just be the value of `expect_proxy`, not a separate property.

Reply:
done